src/utils.rs

```diff
@@ -1,16 +1,30 @@
 use gettextrs::gettext;
 use gtk::glib;
+use lazy_static::lazy_static;
+use regex::Regex;
 use std::future::Future;
 use tdgrand::enums::MessageContent as TelegramMessageContent;
 
 use crate::RUNTIME;
 
+lazy_static! {
```

Comment:
I'd use once_cell::Lazy instead of lazy_static, like I do [here](https://github.com/melix99/telegrand/blob/778effa6a2375a8b906dc99a81ae3b2f4a41a5db/src/main.rs#L19).